lsp-pyright.el

```diff
@@ -113,15 +119,20 @@ set as `python3' to let ms-pyls use python 3 environments."
   :type 'string
   :group 'lsp-python-ms)
 
-;; taken from lsp-python-ms
+(defcustom lsp-pyright-venv nil
+  "Path to folder with subdirectories that contain virtual environments."
+  :type 'string
+  :group 'lsp-python-ms)
```

Comment:
Nits: `lsp-python-ms` -> `lsp-pyright`

Although not related to this PR: `lsp-pyright-python-executable-cmd` also have incorrect group (my bad).

Reply:
will fix them.